docs/source/yelpsoa_configs.rst

```diff
@@ -17,6 +17,11 @@ specify the following options:
     available free CPU, but is guaranteed to get the CPU shares specified.  For
     a more detailed read on how this works in practice, see the docs on `isolation <isolation.html>`_.
 
+    * ``cpu_burst_add``: Number of additional CPUs an instance may use while
```

Comment:
"Maximum number", to make it clear this is a hard limit?

Reply:
Okay, I'll make the change.